src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Core/Module.ts

```diff
@@ -86,6 +88,7 @@ export var register = (angular, config, metaApi) => {
             AdhCrossWindowMessagingModule.moduleName,
             AdhEmbedModule.moduleName,
             AdhMetaApiModule.moduleName,
+            AdhNamesModule.moduleName,
```

Comment:
This should not be required. (Given that we add the correct dependency everywhere this module is used, including in templates)